RELEASING.md

```diff
@@ -0,0 +1,17 @@
+# HOW TO PUBLISH PROJECT:
+
+The SLDS Validator for Visual Studio Code gets published to the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforce-vscode-slds), where it
+is downloaded by users.
+
+## Update Validator
+
+1. Make code changes
+1. Update semver in `package.json` as appropriate
+1. Run `npm install` to update `package-lock.json` with new semver
```

Comment:
I wonder if we should make a note about updating the changelog as a part of the code changes when updating `package.json`

Also we can add a note about creating a git tag once changes are merged in. 

The steps might look something like:

1. Pull down the latest master
1. Create a git tag with `git tag vMajor.Minor.Patch`
1. Push up the tag `git push --tags` 
1. Create a Github release with the newly pushed tag here: https://github.com/forcedotcom/salesforcedx-vscode-slds/releases/new
1. Name the release as `Release vMajor.Minor.Patch` and add the contents of the changelog as the release description.